### PR TITLE
paladin: bump version

### PR DESCRIPTION
**Pull request by kenmays**
Paladin 2.8~git built on hrev54460 x86_gcc2 and x86_64.
Closes: https://github.com/haikuports/haikuports/issues/5168

### CODE REVIEW

**Comment by extrowerk** [comment]
“ NOTE: Encountered 'find pcre2.h' bug through haikuporter x86 build - but build and tests works.”

What the heck does this even mean? You could supply the build log, but no, let just them pull your pr manually, and build it also manually, just to find it out...

So some kind of regex stuff in it needs pcre, but the build system was unable to find the required header, but also this functionality just optional, so the build finished?
But what functionality needs this? Does it affect the usability seriously? Or not?
Btw, i am almost sure we dont want 2 different version (gcc2, gcc8) for x86.
I know this was the fastest and easiest way to accomplish it, but there *MUST* be a better way for this.
Maybe we should disable gcc2 arch, idk, but i would hate myself to tell to users “Just install the other version!”
And what will happen if both package installed in the same time? I see no safe, sane and clear way to do this.

**Comment by kenmays** [comment]
@extrowerk - For Paladin:
1. Is the point of Paladin to support GCC 2.x and x86/x86_64 GCC8? Why not ask the developer?
2. pcre.h issue was on my system... but is that also on build system or elsewhere? Any old build logs to review?

**Comment by extrowerk** [comment]
>Is the point of Paladin to support GCC 2.x and x86/x86_64 GCC8? Why not ask the developer?

A nicely written program can be compiled with both compiler So the recipe-creator are responsible to answer the question why 2 different version needed. And thats you, not the developer.

>pcre.h issue was on my system... but is that also on build system or elsewhere? Any old build logs to review?

You meant only with the freshly introduced secondary arch have problems with pcre, secondary arch was not available before, so no build logs.

File: haiku-apps/paladin/paladin-2.7.recipe [hunk 1]
```diff
@@ -40,22 +41,23 @@ PROVIDES="
 	app:SymbolFinder
 	"
 REQUIRES="
-	haiku
-	lib:libpcre
-	lib:libz
+	haiku$secondaryArchSuffix
+	lib:libpcre$secondaryArchSuffix
+	lib:libunittest++$secondaryArchSuffix
```
[the remark this answers is not in the document]
Comment (kenmays) [reply haiku-apps/paladin/paladin-2.7.recipe:46]: Used during testing (buildscript).

[the remark this answers is not in the document]
Comment (extrowerk) [reply haiku-apps/paladin/paladin-2.7.recipe:46]: This section is the runtime deps for Paladin. Does it really depends on it?
Run `objdump -x /path/to/Paladin_executable | grep NEEDED` on the generated Paladin executable and post the results here.

**Comment by Begasus** [comment]
Issue reported here: https://github.com/haikuports/haikuports/issues/5198
Maybe it  would good to investigate why this can't be build for gcc2

**Comment by Begasus** [comment]
Build still fails with gcc2: https://paste.ubuntu.com/p/YPfQTXsdtp/

**Comment by kenmays** [comment]
Paladin 2.7 can build/work on x86_gcc2 and x86. Paladin 2.8~git (as of 20200722) has build issues on x86_gcc2, but built and works fine on x86_64.

**Comment by kenmays** [comment]
Fixed gcc2 build. All testing passed.

**Comment by Begasus** [comment]
So far there haven't been any rejection on merging this PR, if not I'll merge in a short time ...

**Pull request merged**